src/os/ui/config/betatoggleui.js

```diff
@@ -0,0 +1,101 @@
+goog.declareModuleId('os.file.BetaToggleUI');
+
+import Settings from '../../config/settings.js';
+import {ROOT} from '../../os.js';
+import Module from '../module.js';
+
+
+
+/**
+ * The column mapping settings UI directive
+ * @return {angular.Directive}
+ */
+export const directive = () => ({
+  restrict: 'E',
+  replace: true,
+  templateUrl: ROOT + 'views/file/betatoggle.html',
+  controller: Controller,
+  controllerAs: 'ctrl'
+});
+
+/**
+ * The element tag for the directive.
+ * @type {string}
+ */
+export const directiveTag = 'betatoggle';
+
+
+Module.directive(directiveTag, [directive]);
+
+
+/**
+ * Controller for beta features settings.
+ * @unrestricted
+ */
+export class Controller {
+  /**
+   * Constructor.
+   * @param {!angular.Scope} $scope
+   * @ngInject
+   */
+  constructor($scope) {
+    /**
+     * @type {?angular.Scope}
+     * @protected
+     */
+    this.scope = $scope;
+    this.Settings = Settings.getInstance();
+    // set up an array to store all beta features
+    this['betaFeatureNames'] = [];
+    // iterate through all beta features in the settings.json
+    for (const key in this.Settings.get('betaFeatures')) {
```

Comment:
(nit) I'd made betaFeatures a global or constant somewhere at the top of this file (outside of the class). The syntax usually looks like this
`const export BETA_FEATURES_PATH = 'betaFeatures'`

then you can use that where ever you need instead of hardcoding it in the file. Then if you ever need to change the path, you only update it in one place.